book/src/help_bn.md

```diff
@@ -385,6 +385,9 @@ Options:
           Number of validators per chunk stored on disk.
       --slots-per-restore-point <SLOT_COUNT>
           DEPRECATED. This flag has no effect.
+      --state-cache-headroom <N>
+          Minimum number of states to cull from the state cache when it gets
+          full [default: 1]
```

Comment:
Should this flag be hidden?

Reply:
I don't mind it not being hidden. Or maybe we hide it until we implement https://github.com/sigp/lighthouse/issues/7135